bin/appliance_console

```diff
@@ -280,7 +280,7 @@ Static Network Configuration
 
         when 'hostname'
           say("Hostname Configuration\n\n")
-          new_host = just_ask("new hostname", host)
+          new_host = just_ask("new hostname", host, HOSTNAME_REGEXP)
```

Comment:
Is there an error message that would be more helpful than "In the specified format"?

Reply:
Sure, whatcha think about ```"Please provide a text string up to 24 characters drawn from the alphabet (A-Z), digits (0-9), minus sign (-), and period (.) per IETF requirements."``` ?

Reply:
Also, since I'm gone for two weeks, any chance you'd be amenable to merging this and letting me do the error text update in a follow up when I get back, please?

Reply:
um yea, think it is fine the way it is.

Reply:
> Sure, whatcha think about `"Please provide a text string up to 24 characters drawn from the alphabet (A-Z), digits (0-9), minus sign (-), and period (.) per IETF requirements."` ?

Maybe "Please enter a valid hostname"?

Reply:
I like Brandon's suggestion.

Either that or:

>Please pick a valid host name according to the IETF standard and not something esoteric like having a space or other non alphanumeric character. Sure, minus signs are acceptable and if you want to fully qualify it then by all means add a period. But if you are going to do that then please put in a proper TLD (e.g. .com) and don't put in some bogus string that will puke when we send it over to dns. Thanks